controllers/etcdpeer_controller.go

```diff
@@ -78,6 +78,16 @@ func bindAllAddress(port int) *url.URL {
 	}
 }
 
+func clusterStateValue(cs etcdv1alpha1.InitialClusterState) string {
+	if cs == etcdv1alpha1.InitialClusterStateNew {
+		return "new"
+	} else if cs == etcdv1alpha1.InitialClusterStateExisting {
+		return "existing"
+	} else {
+		return ""
+	}
+}
+
```

Comment:
We shouldn't allow `""` here,right?
Since `--initial-cluster-state` is not documented as having a default value, I think we should make `EtcdPeer.Spec.Bootstrap.InitialClusterState` a required API field.

```
clustering flags:

	--initial-advertise-peer-urls 'http://localhost:2380'
		list of this member's peer URLs to advertise to the rest of the cluster.
	--initial-cluster 'default=http://localhost:2380'
		initial cluster configuration for bootstrapping.
	--initial-cluster-state 'new'
		initial cluster state ('new' or 'existing').

```

Reply:
I think it is a required API field. We could throw an error in the fallthrough case.